src/diffusers/models/modeling_utils.py

```diff
@@ -783,6 +785,71 @@ def num_parameters(self, only_trainable: bool = False, exclude_embeddings: bool
         else:
             return sum(p.numel() for p in self.parameters() if p.requires_grad or not only_trainable)
 
+    def _convert_deprecated_attention_blocks(self):
```

Comment:
@patrickvonplaten Is it ok to put all of the conversion code for each class in this one method and determine which model it's in by checking self.__class__.__name__? I think it's a bit more straightforward than splitting it up in the different model definitions